code/modules/mob/living/living.dm

```diff
@@ -773,6 +773,12 @@ Sorry Giacom. Please don't be mad :(
 		step_towards(src,S)
 
 /mob/living/narsie_act()
+	if(is_servant_of_ratvar(src) && !stat)
+		src << "<span class='userdanger'>You resist Nar-Sie's influence... but not all of it. <i>Run!</i></span>"
+		adjustBruteLoss(35)
+		if(reagents)
+			reagents.add_reagent("heparin", 5)
+		return 0
```

Comment:
What Anonmare said. I'm probably going to be implementing the cult v. cult mechanics and submode in this PR as well.